testdata/xml-data/test-data-systematic.xml

```diff
@@ -5,8 +5,18 @@
        xsi:schemaLocation="https://dasch.swiss/schema ../../src/dsp_tools/resources/schema/data.xsd"
        shortcode="4123"
        default-ontology="testonto">
+    
+
+    <authorship id="authorship_1">
+        <author>
+            Cavanagh, Annie
+        </author>
+    </authorship>
+    <authorship id="authorship_2">
+        <author>Johannes Nussbaum</author>
+    </authorship>
+
```

Comment:
I admit that mixing `<authorship>` and `<permissions>` tags is a bit messy, but I wanted to have it tested...